subprojects/build-comparison/src/main/java/org/gradle/api/plugins/buildcomparison/gradle/internal/DefaultGradleBuildInvocationSpec.java

```diff
@@ -34,6 +35,7 @@
     private List<String> tasks = new LinkedList<String>();
     private List<String> arguments = new LinkedList<String>();
 
+    @Inject
```

Comment:
It's part of the contract of `ObjectFactory.newInstance()`: it uses JSR-330 semantics for dependency injection, which requires an `@Inject` annotation on any constructor that accepts parameters.